sql/core/src/test/resources/sql-tests/results/ansi/date.sql.out

```diff
@@ -268,7 +268,8 @@ struct<>
 -- !query output
 org.apache.spark.SparkIllegalArgumentException
 {
-  "errorClass" : "_LEGACY_ERROR_TEMP_3209",
+  "errorClass" : "INTERVAL_ERROR.ILLEGAL_DAY_OF_WEEK",
```

Comment:
Could you explain how this error related to the type `INTERVAL`? Let's move `ILLEGAL_DAY_OF_WEEK` out of `INTERVAL_ERROR`.